### DISCUSSION TITLE
Варнинг ListMinus

### DISCUSSION THREAD

**Discussion by Spyker086** [totumonline/totum-ru-issues-and-discussions#475] [ОШБК!] [answered]
Добрый день!
Имеем такой список: 
[
  "",
  "",
  "",
  "",
  "",
  "",
  "",
  "",
  [
    "12",
    "160чел/2гр",
    "-",
    "61"
  ],
  "",
  [
    "36",
    "3зан",
    "-",
    "63"
  ],
  ""
]

Когда к нему применяю listMinus(list: $ls; item: "") , чтобы удалить пустые элементы, получаю варнинг тотума:
Array to string conversion

В результате ListMinus отрабатывает. Может кто сталкивался? Как исключить этот варнинг?

**Comment by Spyker086** [comment]
Отбой решил ListFilterом

**Comment by Spyker086** [comment, marked as answer]
Отбой решил ListFilterом

**Reply by ttm-alexeypolunin** [reply]
Дело в то, что listMinus работает с одним уровнем только, а здесь вложенные списки. Поэтому он и не может с ними ничего сделать.

**Reply by Spyker086** [reply]
Фича это или баг вам виднее, но в результате ListMinus выдает корректный результат, только через варнинг
